### PR TITLE
[Synapse] - New-AzSynapseFirewallRule correct IP ranges for -AllowAllAzureIp and -AllowAllIp 

### PR DESCRIPTION
**Pull request by idear1203**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/882
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Synapse/Synapse.Test/ScenarioTests/FirewallTests.ps1 [hunk 1]
```diff
@@ -48,6 +48,15 @@ function Test-SynapseFirewall
...
         Assert-True {Remove-AzSynapseFirewallRule -ResourceGroupName $resourceGroupName -WorkspaceName $workspaceName -Name $firewallRuleName -PassThru -Force} "Remove firewall rule failed"
+
+		# create firewall rule to allow all Azure IP
+		$firewallCreated = New-AzSynapseFirewallRule -ResourceGroupName $resourceGroupName -WorkspaceName $workspaceName -AllowAllAzureIp
+        Assert-AreEqual "0.0.0.0" $firewallCreated.StartIpAddress
+        Assert-AreEqual "0.0.0.0" $firewallCreated.EndIpAddress
+
+		$firewallCreated = New-AzSynapseFirewallRule -ResourceGroupName $resourceGroupName -WorkspaceName $workspaceName -AllowAllIp
+        Assert-AreEqual "0.0.0.0" $firewallCreated.StartIpAddress
+        Assert-AreEqual "255.255.255.255" $firewallCreated.EndIpAddress
```
Comment (BethanyZhou) [inline src/Synapse/Synapse.Test/ScenarioTests/FirewallTests.ps1:59]:
```suggestion
	# create firewall rule to allow all Azure IP
	$firewallCreated = New-AzSynapseFirewallRule -ResourceGroupName $resourceGroupName -WorkspaceName $workspaceName -AllowAllAzureIp
        Assert-AreEqual "0.0.0.0" $firewallCreated.StartIpAddress
        Assert-AreEqual "0.0.0.0" $firewallCreated.EndIpAddress

	$firewallCreated = New-AzSynapseFirewallRule -ResourceGroupName $resourceGroupName -WorkspaceName $workspaceName -AllowAllIp
        Assert-AreEqual "0.0.0.0" $firewallCreated.StartIpAddress
        Assert-AreEqual "255.255.255.255" $firewallCreated.EndIpAddress
```

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
LGTM, leave one comment

File: src/Synapse/Synapse.Test/ScenarioTests/FirewallTests.ps1 (hunk 1 above)
Comment (idear1203) [reply src/Synapse/Synapse.Test/ScenarioTests/FirewallTests.ps1]: Thanks for the guide. I changed the tab to spaces and added breaking change suppressions.

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Pull request merged**